eclipse-jdt/eclipse.jdt.ui
# Add number of changed files to PreviewWizardPage

Add number of changed files to PreviewWizardPagel
bump pom file version
Use number of files within existing description
use two strings for single and multi case instead of one

## What it does
Instead of (currently) presenting this text after cleanups have been searching for hits:

The following changes are necessary to perform the refactoring.

you get this text:

The following change to 1 file is necessary to perform the refactoring.

or 

The following changes to {0} files are necessary to perform the refactoring.

That way you are able to see a difference even in case of a very huge number of hits when changeing the selected cleanups.
Otherwise you get a list presented but for hundreds of hits it is impossible to tell the number of hits increased or decreased.

## How to test
Use the context menu for cleanup on a java project and play with the selection of cleanups. You should now see the number of hits immediately in the preview presented.

## Author checklist

- I have thoroughly tested my changes
- The change is following the [coding conventions](https://wiki.eclipse.org/Platform/How_to_Contribute#Coding_Conventions)
- I have signed the [Eclipse Contributor Agreement (ECA)](https://www.eclipse.org/legal/ECA.php)
